hs_core/models.py

```diff
@@ -1523,7 +1523,8 @@ class Bags(models.Model):
     content_type = models.ForeignKey(ContentType)
 
     content_object = generic.GenericForeignKey('content_type', 'object_id')
-    bag = models.FileField(upload_to='bags', max_length=500, storage=IrodsStorage() if getattr(settings,'USE_IRODS', False) else DefaultStorage(), null=True) # actually never null
+    #bag = models.FileField(upload_to='bags', max_length=500, storage=IrodsStorage() if getattr(settings,'USE_IRODS', False) else DefaultStorage(), null=True) # actually never null
+    bag_url = models.URLField(null=True, blank=True)
```

Comment:
The bag_url is added here to provide download path for a specific resource when user clicks on "Download all" from resource landing page. It used to be provided as part of IrodsStorage. Now that we don't need to upload bags from Django to iRODS, we don't need bag upload FileField that ties to IrodsStorage backend storage engine. So I am storing it here for purpose of downloading the specific resource. This URL has a value pattern of "/django_irods/download/path=bagname.zip" without absolute URL encoded, so I don't see an issue with it. What other better ways to achieve this purpose if any?